src/Symfony/Component/ClassLoader/ApcClassLoader.php

```diff
@@ -118,11 +118,13 @@ public function loadClass($class)
      *
      * @param string $class A class name to resolve to file
      *
-     * @return string|null
+     * @return string|false
```

Comment:
This does not seem correct. The other class loaders have `string|null`. As they are decorated by ApcClassLoader something is wrong. If they do not return false at all, then this whole change seems irrelevant.